lib/native/auth.spec.ts

```diff
@@ -5,6 +5,8 @@
 
 import { auth as native, http as native_http } from '../index';
 import { aws_sign_request } from './auth';
+import { InputStream } from './io';
+import { PassThrough } from "stream";
```

Comment:
trivial: still necessary? These are the only changes in the file

Reply:
They are used by the tests